From b05b004192f9922c90a4399bee5c1cacc5a48b67 Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Mon, 9 Jan 2023 19:50:39 +0100
Subject: [PATCH] Switch to correct build-system definition

pyproject.toml:
Use the correct PEP517 build-backend, as defined by poetry upstream
(https://python-poetry.org/docs/pyproject#poetry-and-pep-517).
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 709adf1..d3292af 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,5 +31,5 @@ matrix_sso_helper = ["aiohttp"]
 matrix_upload = ["python-magic", "requests"]
 
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
-- 
2.39.0

